packages/did-core-test-server/suites/did-spec/did-core-properties.js

```diff
@@ -0,0 +1,382 @@
+const utils = require('./utils');
+const {
+  isValidDID, isValidURI, isValidBase58, isValidJwk, isValidVerificationMethod,
+  getAbsoluteDIDURL
+} = utils;
+const jose = require('node-jose');
+
+const jsonMediaTypes = ['application/did+ld+json', 'application/did+json'];
```

Comment:
What does this do? See comment below

Reply:
https://github.com/w3c/did-test-suite/pull/38#discussion_r601585709